pkgs/servers/polaris/web.nix

```diff
@@ -1,65 +1,31 @@
 { lib
-, stdenv
-, pkgs
+, buildNpmPackage
 , fetchFromGitHub
-, nodejs
-, cypress
+, nodePackages
+, python3
 }:
 
-stdenv.mkDerivation rec {
+buildNpmPackage rec {
   pname = "polaris-web";
-  version = "build-55";
+  version = "67";
 
   src = fetchFromGitHub {
     owner = "agersant";
     repo = "polaris-web";
-    rev = version;
-    sha256 = "2XqU4sExF7Or7RxpOK2XU9APtBujfPhM/VkOLKVDvF4=";
+    rev = "build-${version}";
+    hash = "sha256-mhrgHNbqxLhhLWP4eu1A3ytrx9Q3X0EESL2LuTfgsBE=";
   };
 
+  npmDepsHash = "sha256-lScXbxkJiRq5LLFkoz5oZsmKz8I/t1rZJVonfct9r+0=";
+
   nativeBuildInputs = [
-    nodejs
+    nodePackages.node-gyp-build
```

Comment:
@lilyinstarlight Is this the way to go despite using `buildNpmPackage`?

Reply:
I think we should probably package node-gyp-build separately so that we do not rely on nodePackages anymore

Reply:
If the package depends on `node-gyp-build`, it should be in the `devDependencies` in `package.json` (but `node-gyp` itself is included in the nodejs distribution and should not be included)

Does it not do that?

Reply:
Oh I admittedly had to look up what node-gyp-build was again, but this is the thing that basically checks whether prebuilt binaries exist in the downloaded package and runs its own `node-gyp rebuild` if not

I'm wondering if we should just make a fake `node-gyp-build` command that just does `node-gyp rebuild`...